Packages/com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -306,6 +306,7 @@ public Color clearColor
         internal bool isBlitRenderPass { get; set; }
 
         internal bool useNativeRenderPass { get; set; }
+        internal bool isNativeRenderPass = false; // To avoid Y-flip when NRP is used but the pass does not support NRP.
```

Comment:
Is this a global setting? Can we query the setting instead of storing it inside every RenderPass?

Reply:
This is for the finalBlitPass's corner case. It won't be able to check the global settings insider there.